app/build.gradle

```diff
@@ -5,7 +5,7 @@ android {
     buildToolsVersion '26.0.2'
     defaultConfig {
         applicationId "gr.escsoft.michaelprimez.searchablespinnerexamples"
-        minSdkVersion 21
+        minSdkVersion 19
```

Comment:
Why 19? Please use minSdkVersion 15 and try run it in an emulator. You'll see some fixes you have to make related to ViewTreeObserver, actually the app is crashing with "java.lang.NoSuchMethodError: android.view.ViewTreeObserver.removeOnGlobalLayoutListener". Also to support older version than API 17 you should add paddingRight, paddingLeft whenever a paddingStart, paddingEnd is used.

Reply:
Hey I did make the library minimum sdk 15. If u need to make the demo app suitable for <19 then please make the needed changes. I am using the lib in one of my target projects and it's working fine over there. The pr purpose was to make the lib support sdk as low as 15

Reply:
And what you did about the "NoSuchMethodError"? did you remove it? And what you did about paddingStart, paddingEnd? Did you fix it in the instance of the library that you use in your projects? If yes, why don't you fix it in the pr in order to help others? It is not so polite to make an uncompleted pr and then ask for other to make fixes. Probably in your projects you don't use min sdk 15 but the title of the pr is "Min sdk15" after all. I don't understand you at all...